proto/tendermint/rpc/grpc/types.proto

```diff
@@ -23,10 +45,43 @@ message ResponseBroadcastTx {
   tendermint.abci.ResponseDeliverTx deliver_tx = 2;
 }
 
+message BlockByHashResponse {
+  tendermint.types.Block block = 1;
+}
+
+message BlockByHeightResponse {
+  tendermint.types.Block block = 1;
+}
+
+message CommitResponse {
+  int64                    height                = 1;
+  int32                    round                 = 2;
+  tendermint.types.BlockID block_id              = 3 [(gogoproto.nullable) = false, (gogoproto.customname) = "BlockID"];
+  repeated tendermint.types.CommitSig signatures = 4 [(gogoproto.nullable) = false];
+}
+
+message ValidatorSetResponse {
+  repeated tendermint.types.Validator validators         = 1;
+  tendermint.types.Validator          proposer           = 2;
+  int64                               total_voting_power = 3;
```

Comment:
voting power is not needed actually and is not provided by original rpc (private field)

Reply:
do I need to remove it? or we can leave it just in case? I'm fine with any